File: Baekjoon/python/2581.py
num_list = []
    
def f_2581(M, N):
    for i in range(M, N + 1):
        count = 0
        if i > 1:
            for j in range(2, i):
                if i % j == 0:
                    count += 1
                    break
            if count == 0:
                num_list.append(i)

            
    if len(num_list) != 0:
        return(sum(num_list), num_list[0])
    else:
        return("-1")

File: Baekjoon/C++/2581.cpp
#include <iostream>
using namespace std;

int main(){
	int m, n;
	int sum = 0;
	int min = -1;
	int count = 0;

	cin >> m >> n;

	for (int i = m; i <= n; i++){
		for (int div = 1; div <= i; div++){
			if (i % div == 0)
				count++;
		}
		if (count == 2){
			if (min == -1)
				min = i;
			sum += i;
		}
		count = 0;
	}
	if (min == -1)
		cout << -1 << endl;
	else
		cout << sum << endl << min << endl;
}
